src/Bicep.LangServer.IntegrationTests/CompletionTests.cs

```diff
@@ -304,6 +304,61 @@ public async Task Completions_are_not_offered_immediately_after_open_paren()
             await RunCompletionScenarioTest(this.TestContext, ServerWithBuiltInTypes, fileWithCursors, AssertAllCompletionsEmpty, '|');
         }
 
+        [TestMethod]
+        public async Task Parameter_type_description_should_be_shown_for_bicep_symbol_completions()
+        {
+            var bicepTextWithCursor = @"
+@allowed(
+    [
+        0
+        1
+    ]
+)
+@description('this is an int value')
+param myInt int
+
+@allowed(
+    [
+        'value1'
+        'value2'
+    ]
+)
+@description('this is a string value')
+param myStr string
+
+@description('this is a bool value')
+param myBool bool
+
+resource storageAct 'Microsoft.Storage/storageAccounts@2022-09-01' = {
+  name: |
+}
+";
+
+            await RunCompletionScenarioTest(this.TestContext, DefaultServer, bicepTextWithCursor, completions => {
+              
+              completions.Count().Should().Be(1);
+              var symbolCompletions = completions.First().Items.Where(x => x.Kind == CompletionItemKind.Field);
+
+              symbolCompletions.Should().SatisfyRespectively(
```

Comment:
Could you also add assertions for non-params symbols?
`var`, `resource`, `module` (`outputs` won't be shown in completions)